src/style.css

```
@@ -0,0 +1,132 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 198px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+
+  margin: 0 auto;
+
+  height: 134px;
+  width: 160px;
+}
+
+.card__name {
+  margin: 34px 0 0;
```

Can you add just margin top?

```suggestion
  margin-top: 34px;
```